config/samples/cxx_example_loadtest.yaml

```diff
@@ -50,14 +50,70 @@ spec:
   # results:
   #   bigQueryTable: "example-project.foo.demo_dataset"
 
-  # Scenarios are separate objects under this proposal. While
-  # they specify configuration for clients and servers, they are
-  # highly specialized. I doubt we want to modify controller logic
-  # whenever the Scenario proto is changed. This is open to
-  # discussion, however.
-  #
-  # These can be stored in ConfigMaps or they can be high-level
-  # Kubernetes objects themselves. If we did want to couple, they
-  # could also be inlined.
-  scenarios:
-    - name: cxx-example-scenario
+  # ScenariosJSON is string with the contents of a Scenarios message, formatted
+  # as JSON. See the Scenarios protobuf definition for details:
+  # https://github.com/grpc/grpc-proto/blob/master/grpc/testing/control.proto.
+  scenariosJSON: |
+    {
+      "scenarios": [
+        {
+          "name": "cpp_generic_async_streaming_ping_pong_secure",
```

Comment:
nit: add a note that the exact contents of this JSON were generated by the scenario_config.py file?

Reply:
qq: what will be the name of the scenario that will end up in bigQuery if I choose to upload the results? Will it be "cpp_generic_async_streaming_ping_pong_secure"? (that's what I would expect and also AFAIR that's what the legacy system does). 
I think there should be a comment that mentions that (if seems quite important).

Reply:
Yes, the scenario name will be used in BigQuery, since uploads is a responsibility of the driver.